src/ethereum/arrow_glacier/vm/instructions/block.py

```diff
@@ -46,13 +46,13 @@ def block_hash(evm: Evm) -> None:
         # Default hash to 0, if the block of interest is not yet on the chain
         # (including the block which has the current executing transaction),
         # or if the block's age is more than 256.
-        hash = b"\x00"
+        b_hash = b"\x00"
```

Comment:
The "b" should be expanded, but not to `block_hash` or else it'll overwrite the function.